sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -2498,6 +2498,13 @@ class Dataset[T] private[sql](
    *
    * Use [[summary]] for expanded statistics and control over which statistics to compute.
    *
+   * NOTES:
+   *  - String columns are countable, so count is accurate
+   *  - String columns are comparable, so min/max are accurate
```

Comment:
I don't understand what both mean. Strings are already countable and comparable.

Reply:
yes, maybe I just over explained